File: pythonExercise/LibraryManagementSystem/Book.py
class Book:
    def __init__(self, title, author, isbn):
        self._title = title
        self._author = author
        self._isbn = isbn
        self._availability = True  

    # Getters
    def get_title(self):
        return self._title

    def get_author(self):
        return self._author

    def get_isbn(self):
        return self._isbn

    def is_available(self):
        return self._availability

    # Setter for availability
    def set_availability(self, status):
        self._availability = status

File: JavaExercise/LibraryMangementSystem/Book.java
package org.example.Libraryy;

//BOOK class
public class Book {
    private String title;
    private String author;
    private String isbn;
    private boolean available;

    public Book(String title, String author, String isbn) {
        this.title = title;
        this.author = author;
        this.isbn = isbn;
        this.available = true;
    }

    // Gettermethods
    public String getTitle() {
        return title;
    }
    public String getAuthor() {
        return author;
    }
    public String getIsbn() {
        return isbn;
    }
    public boolean isAvailable() {
        return available;
    }

    // Settermethod
    public void setAvailable(boolean available) {
        this.available = available;
    }
}

